clarity/contracts/vaults-v2/arkadiko-vaults-operations-v1-1.clar

```diff
@@ -119,12 +119,14 @@
 )
   (let (
     (owner tx-sender)
-    (nicr (/ (* collateral u100000000) debt))
-    (collateral-info (unwrap! (contract-call? vaults-tokens get-token (contract-of token)) (err ERR_UNKNOWN_TOKEN)))
     (stability-fee (try! (contract-call? vaults-helpers get-stability-fee vaults-tokens vaults-data owner (contract-of token))))
+    (new-debt (+ debt stability-fee))
+
+    (nicr (/ (* collateral u100000000) new-debt))
```

Comment:
Q: isn't this gonna result in issues for the positioning in the off-chain vaults API? the calculation method has changed? Or is this already incorporated in the vaults API?

Reply:
A: The FE will need to get the current stability fees, add these to the debt amount and use the total to call the hints API. Similar to the redemption script.